CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 cmake_minimum_required(VERSION 2.8)
 project(fluent-bit)
 
+set(CMAKE_POSITION_INDEPENDENT_CODE ON)
+
 # Update CFLAGS
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -fPIC ")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall ")
```

Comment:
MSVC also cannot understand `-Wall` compiler option.
We should use `/Wall` instead.
https://msdn.microsoft.com/ja-jp/library/thxezb7y.aspx

Reply:
Hmm, it's reasonable.